java/poi/src/test/java/com/example/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
Y tho?

Reply:
:laughing:  This a placeholder test file maven created during project initialization.